alpa/device_mesh.py

```diff
@@ -908,6 +1009,9 @@ def _launch_xla_servers(self):
                 # "NCCL_LAUNCH_MODE": "PARALLEL",
                 # "XLA_FLAGS": "--xla_dump_to=hlo --xla_dump_hlo_pass_re=.*"
                 # "NCCL_DEBUG": "INFO" if i == 0 else "VERSION",
+                # "NCCL_DEBUG_SUBSYS": "ALL",
+                "NCCL_ALGO": "Ring",
```

Comment:
Revert these changes. If they are required for broadcast, only set the when `global_config.resharding_mode == "broadcast"`

Reply:
OK